docs/developers/setting-up-metamask.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Setting up Metamask
+sidebar_position: 4
+description: Guide on setting up Metamask for development purposes
+keywords:
+  - metamask
+  - subspace network
+---
+
+### Adding Subspace RPC using Metamask
+---
+You can use any EVM-compatible wallet, this guide will be using [**Metamask**](https://metamask.io/) wallet.
+
+First, let’s set up a MetaMask wallet and then go over connecting it to the Subspace Development Network.
+
+1. Go to [Metamask Website](https://metamask.io/) and download an extension for your browser.
+   Select your preferred language at the top-right corner. 
+   Read and agree to MetaMask terms of use.
+
+  ![Metamask-1](/img/developers/MetaMask-1.png)
+
+
+2. Click on “Create a new wallet”
```

Comment:
Missing full stop or formatting is not being rendered quite right.

Reply:
Added a full stop. I think it's okay for the second sentence to start on the same line as the beginning of a paragraph, so no changes to that.